modules/unsupported/mbstyle/src/test/java/org/geotools/mbstyle/transform/VisualTransformerTest.java

```diff
@@ -108,6 +111,11 @@ public void setUp() throws Exception {
         lineZigFS = ds.getFeatureSource("testlinezigs");
         pointsWithMarksFS = ds.getFeatureSource("testmarks");
         bounds = new ReferencedEnvelope(0, 10, 0, 10, CRS.decode("EPSG:4326"));
+        
+        
+        FontCache.getDefaultInstance().registerFont(
+        Font.createFont(Font.TRUETYPE_FONT, TestData.getResource(LabelShieldTest.class, "Vera.ttf")
+        		 .openStream()));
```

Comment:
So, looks like there are some additional steps to get the fonts working, good to know.

Reply:
Jody helped me through here, also of note as you may have seen in the community emails vera is the go to for any visual tests going forward as well.